### DISCUSSION TITLE
Sources not reporting all conversions

### DISCUSSION THREAD

**Discussion by webtail** [plausible/analytics#2299] [Anything Else]
Any idea how this is possible? 6 conversions/orders but top sources is only reporting 2 of them?

![01](https://user-images.githubusercontent.com/43131449/193693977-2456609d-c2a7-4817-a7dd-7964f84833f2.jpg)

Thxs

**Comment by metmarkosaric** [comment]
try clicking on the "details" button in the top sources report. there should be direct/none traffic there too

**Reply by webtail** [reply]
That's it, thxs @metmarkosaric !
